lib/scripts/behaviour.js

```diff
@@ -81,6 +82,18 @@ var dw_behaviour = {
         }
     },
 
+    /**
+     * Display confirm dialog on page restore action
+     */
+    pageRestoreConfirm: function(){
+        var $revert = jQuery('#dokuwiki__pagetools li.revert a');
+        if($revert.length) {
```

Comment:
Technically you don't need this length check because jQuery will simply ignore the `on` call if there is nothing in the selector. (I know we have some old scripts not written in an optimized way...)

Reply:
Removed ``.length`` property in last commit.

> (I know we have some old scripts not written in an optimized way...)

I'll do a check in the DW (and my) scripts to see if they need to be optimized.